pyshader/_generator_bc.py

```diff
@@ -135,8 +136,46 @@ def _get_label_id(self, label_value):
             self._labels[label_value] = label_id
         return self._labels[label_value]
 
+    def errinfo(self, *variables):
+        """Get error info for the current moment during compiling
+        (source filename and line number) and including the names of the given
+        variables.
+        """
+        filename = getattr(self, "_src_filename", "")
+        linenr = getattr(self, "_src_linenr", 0)  # first line is 1 (not 0)
```

Comment:
arguably you could use the name `row` here instead of `linenr`, its a shorter name and is easier to pronounce?

Reply:
Sure!

Reply:
No, wait. I think the `filename` plus `linenr` combo is more common and thus easier to recognize than using `row`.

Reply:
Though sometimes `lineno` is used ;)